package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "root",
-  "version": "3.24.0",
+  "version": "3.25.0",
```

Comment:
As this is a readme change I'd have said a #trivial PR or patch version would suffice here but happy to approve in any case.

Reply:
@DamianMullins thanks for pointing that out, I didn't realise. Happy to change this, let me know!

Reply:
Probably best to stick with the conventions so updating it would be ideal 👍🏻

Reply:
Hm, I think that updating readme is a #trivial change and should go without version bump as for end users there is no changes, right?

Reply:
Agree, remove the versioning and stick a #trivial prefic in the PR title @adammorr